### ISSUE TITLE
b-pagination does not meet accessibility standards

### ISSUE THREAD

**Issue by C-Tully** [bootstrap-vue/bootstrap-vue#7132]
### Describe the bug

While working on a project with this element in use, the requirement of doing an accessibility scan has pulled up an issue with the ellipse that is used when staggering the visible pages.  

![image](https://user-images.githubusercontent.com/3598136/225062556-8ae76d6b-4d32-4b69-813c-21bda42646f9.png)

### Steps to reproduce the bug

1. Go to [...]
2. Click on [...]
3. Scroll down to [...]

### Expected behavior

Ideally we would be able to either remove the role="separator" or allow the user to overwrite the role/element in question in its entirety, currently utilizing the slot override for #ellipise-text keeps the role on the element and only changes the inner content.

### Versions

**Libraries:**

- BootstrapVue: 2.19.0
- Bootstrap: 4.5.3
- Vue: 2.6.11

### Additional context

Add any other context about the bug here.
